packages/message-relayer/src/service.ts

```diff
@@ -205,16 +216,15 @@ export class MessageRelayerService extends BaseService<MessageRelayerOptions> {
             this.state.lastFinalizedTxHeight,
         })
 
-        const messages = await this._getSentMessages(
-          this.state.lastFinalizedTxHeight,
-          this.state.nextUnfinalizedTxHeight
-        )
-
-        if (messages.length === 0) {
-          this.logger.info('Did not find any L2->L1 messages', {
-            retryAgainInS: Math.floor(this.options.pollingInterval / 1000),
-          })
-        }
+        const messages = (
+          await this._getSentMessages(
```

Comment:
Any reason to nest the `await` here?

Reply:
Refactored slightly to remove this nested `await`.